types.ts

```diff
@@ -0,0 +1,61 @@
+// faucet-pipeline-core types
```

Comment:
I mean, we could move these into faucet-core?

Reply:
Yep, that was the intention here. This is why the file is "split" into two sections. But building the types here, and then moving them into core is easier.